src/index.html

```diff
@@ -22,6 +22,410 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          class="moyo_logo"
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product Image"
+          class="product-card__image"
+        />
+
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-card__product-code">Product code: 195434</p>
+
+        <div class="product-card__rating">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="product-card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="product-card__price">
+          <span class="product-card__label">Price:</span>
+          <span class="product-card__price-tag">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="product-card__button"
+          data-qa="card-hover"
```

Comment:
The `data-qa` attribute is used here for the card button. Ensure that this attribute is consistently used across similar elements if it's intended for testing or automation purposes.